helper/devicex/device.go

```diff
@@ -32,20 +32,22 @@ import (
 	"github.com/shirou/gopsutil/v3/net"
 )
 
-type DeviceI interface {
-	Info() error
-}
-
-var _ DeviceI = new(device)
-
-type device struct {
-	Memory devMemory `json:"memory"`
-	Cpu    devCpu    `json:"cpu"`
-	Net    devNet    `json:"net"`
-	// Disk   devDisk   `json:"disk"`
-}
+type (
+	DeviceI interface {
+		Info() error
+	}
+	device struct {
+		Memory devMemory `json:"memory"`
+		Cpu    devCpu    `json:"cpu"`
+		Net    devNet    `json:"net"`
+		// Disk   devDisk   `json:"disk"`
```

Comment:
Please remove this `// Disk   devDisk   `json:"disk"`` comment

Reply:
ok,i will remove it.